main.go

```diff
@@ -35,6 +35,7 @@ func main() {
 		optimizeParserFlag     = fs.Bool("optimize-parser", false, "generate optimized parser without Debug and Memoize options")
 		recvrNmFlag            = fs.String("receiver-name", "c", "receiver name for the generated methods")
 		noBuildFlag            = fs.Bool("x", false, "do not build, only parse")
+		altEntrypointsFlag     = fs.String("alternate-entrypoints", "", "comma-separated list of rule names that may be used as entrypoints")
```

Comment:
Maybe this flag could be defined as `fs.Var` [see](https://golang.org/src/flag/flag.go#L807), the usage of comma separated list is the actual example for this flag variant.
This would help the cleanup the `main` function, because the code to handle the comma separated list would be on the type, which implements the `flag.Value` interface.

Reply:
Oh nice! Yes, I'll do it this way.